Tests/Remora.Discord.Tests/Samples/Objects/MESSAGE_COMPONENT/CHANNEL_SELECT_COMPONENT.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": 8,
```

Comment:
You should run the scrubber to normalize these samples; after that, this is good to merge.

Reply:
Done!